src/afrolabs/components/kafka.clj

```diff
@@ -14,7 +14,7 @@
    [clojure.string :as str]
    [iapetos.core :as prom]
    [integrant.core :as ig]
-   [java-time :as t]
+   [java-time.api :as t]
```

Comment:
The `java-time` namespace has been deprecated, everywhere I find it I update it to `java-time.api`. This has so far been a zero-risk change.